### DISCUSSION TITLE
[Feature]: add templates for Nestjs project

### DISCUSSION THREAD

**Discussion by ianzone** [web-infra-dev/rsbuild#3408] [General]
### What problem does this feature solve?

wonder if it's more concise to use rsbuild for nestjs projects than rspack. 

### What does the proposed API look like?

add a description page on the framework section.

**Comment by chenjiahan** [comment]
At this stage, Rsbuild is more focused on building web applications. For Node.js applications such as Nest.js, we currently have no plans to provide first-party support in Rsbuild, but I believe this can be achieved through a community-maintained Rsbuild plugin.
